### ISSUE TITLE
Can't drag and drop custom fonts

### ISSUE THREAD

**Issue by theonlyPHiX** [WilliamRagstad/Font-Manager#34]
I wanted to have Neon 80s  ,so I downloaded it from dafont. As maxim(bananagaming) dragged and dropped the zip file I tried to do the same, but it shows like slash within a circle symbol, meaning it can't be dropped there. So I Installed the font and it showed up in the "+" list. But when I click add, jt says the font is not there. I am unable to add custom fonts. It's a great program but can you help me here?

**Comment by WilliamRagstad** [comment]
Are you running Font Manager with administrator privileges?

**Comment by WilliamRagstad** [comment]
This might be a bug for which some characters in the font are not supported at the moment.

**Comment by theonlyPHiX** [comment]
Yes ofc it is running in administrative mode. I tried download an old version still dragging and dropping didn't work. I also installed neon 80s font onto my computer, but when I click add and choose neon 80s it shows some error. It was a path error as much as I remember.

**Comment by OMeguminO** [comment]
It says the path for the font couldn't be found

**Comment by OMeguminO** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/66835737/86523642-7746d700-be6f-11ea-835f-c482af9be42b.png)
